src/wallet/ismine.h

```diff
@@ -14,7 +14,26 @@
 class CWallet;
 class CScript;
 
-/** IsMine() return codes */
+/**
+ * IsMine() return codes, which depend on ScriptPubKeyMan implementation.
+ * Not every ScriptPubKeyMan covers all types, please refer to
+ * doc/release-notes.md#ismine-semantics for better understanding.
```

Comment:
In commit "doc: Add developer documentation to isminetype" (2400c3aadc2e3c87f5a1eda862bbb45213e9898f)

doc/release-notes.md#ismine-semantics doesn't seem to exist

Reply:
https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes.md#ismine-semantics

Reply:
This is https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.21.0.md#ismine-semantics

Reply:
> This is https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.21.0.md#ismine-semantics

fixed!